### PR TITLE
Fix revenue tracking

### PR DESCRIPTION
**Pull request by ccnixon**
Added some final iterations after testing more

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ var Amplitude = module.exports = integration('Amplitude')
   .option('saveParamsReferrerOncePerSession', true)
   .option('deviceIdFromUrlParam', false)
   .option('mapQueryParams', {})
+  .option('trackRevenuePerProduct', true)
```
Comment (hankim813) [inline lib/index.js]: just to play devil's advocate, we sure we want this to default to `true`?

**Review by hankim813 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-integration-amplitude",
   "description": "The Amplitude analytics.js integration.",
-  "version": "2.4.1",
+  "version": "2.5.0",
```
Comment (hankim813) [inline package.json:4]: wondering if all this new feature behavior be a major release?

Comment (ccnixon) [reply package.json:4]: Yeah but it's all still backwards compatible (if we're following SEMVER that's not major)

File: lib/index.js (hunk 1 above)
Comment (ccnixon) [reply lib/index.js]: Crap you're right, that should be false. Good catch

File: package.json (hunk 2 above)
Comment (hankim813) [reply package.json:4]: sounds good!

**Pull request merged**